examples/rpc/rpc-server.cpp

```diff
@@ -81,6 +85,14 @@ static ggml_backend_t create_backend() {
     }
 #endif
 
+#elif GGML_USE_SYCL
```

Comment:
this will give a compile error if you don't remove the `#endif` on line 86

Reply:
Should be resolved now!